src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/CommPkg.cs

```diff
@@ -69,6 +69,11 @@ public CommPkg(
         public bool RfocAccepted { get; set; }
         public ICollection<Certificate> CertificateScopes => _certificateScope;
         // TODO: make CommPkgGuid private after FillGuids has completed
+        // Guid used across modules.
+        // In a system using guid to identify objects between the modules, the guid may identify the object uniquely in one module,
+        // but not uniquely in another module due to many-to-many relationship to objects in that distinct module. 
```

Comment:
this comment should be on top of the class and say something like:

A better name for this class would be InvitationCommPkgRelation or similar.
This entity identifes a CommPkg in relation to an Invitation. Guid is the unique identifier for the relation, while CommPkgGuid specifies which CommPkg the invitation relates to.